Packs/SMB/Integrations/SMB/README.md

```diff
@@ -1,5 +1,6 @@
 <!-- HTML_DOC -->
 <p>Use the SMB integration to upload and download files from an SMB protocol.</p>
+<p>The integration will utilize SMB2 protocol for communication if the remote SMB/CIFS service supports SMB2. Otherwise, it will fallback automatically back to using SMB1 protocol</p>
```

Comment:
The integration will utilize SMB2 protocol for communication if the remote SMB/CIFS service supports SMB2. Otherwise, the integration will automatically fall back to use SMB1 protocol.